builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -262,8 +263,10 @@ func (b *backend) pathUpdateIssuer(ctx context.Context, req *logical.Request, da
 
 	var oldName string
 	if newName != issuer.Name {
+		// If name is updated, we update issuerEntry LastModified?
```

Comment:
I agree this is correct. If we've updated two issuers two cross their names (old-root <-> new root or some such) -- you'd want to inform clients.

Better to make clients fetch the new cert a few extra times than hide changes from them.